frr_exporter.go

```diff
@@ -65,7 +39,14 @@ func main() {
 	logger.Info("Starting frr_exporter", "version", version.Info())
 	logger.Info("Build context", "build_context", version.BuildContext())
 
-	http.Handle(*telemetryPath, handler(logger))
+	nc, err := collector.NewExporter(logger)
+	if err != nil {
+		panic(fmt.Sprintf("Couldn't create collector: %s", err))
```

Comment:
Another apostrophe contraction. Also, `%v` is the generally recommended printf verb for errors.
```suggestion
		panic(fmt.Sprintf("Could not create collector: %v", err))
```